src/e2e-test/features/dataplex/sink/DesignTimeDataplex.feature

```diff
@@ -0,0 +1,137 @@
+@Smoke
+@Regression
+Feature: Dataplex Source - End to end
+
+  @BATCH-TS-DATAPLEX-DSGN-01
+  Scenario: To verify data is not validated successfully when mandatory fields are missing in dataplex sink
+    When Open Datafusion Project to configure pipeline
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "referenceName" with value: "jkhiu"
+    And Click on the Validate button
+
+
+  @BATCH-TS-DATAPLEX-DSGN-02
+  Scenario: To verify user should be able to validate mandatory fields in dataplex sink with BigQuery dataset
+    When Open Datafusion Project to configure pipeline
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "referenceName" with value: "iuhjkhiu"
+    And Enter input plugin property: "location" with value: "us-central1"
+    And Enter input plugin property: "lake" with value: "dp-metastore3-lake"
+    And Enter input plugin property: "zone" with value: "merazone"
+    And Enter input plugin property: "asset" with value: "meraassetbq"
+    And Enter input plugin property: "table" with value: "Tax_table"
+    And Click plugin property: "switch-truncateTable"
+    And Click plugin property: "switch-allowSchemaRelaxation"
+    And Click on the Validate button
+
```

Comment:
Add validation step

Reply:
Which validation step. Do you mean validating input schema?